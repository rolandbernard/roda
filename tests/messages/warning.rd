// test: Test style of warning message
// build: LLVM_PROFILE_FILE="profile/tests/%%p.profraw" ./build/$BUILD/bin/rodac --unknown -U -o --output 2> %.out
