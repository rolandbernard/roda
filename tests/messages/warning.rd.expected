warning[W0001]: --unknown: unknown command line option
warning[W0001]: -U: unknown command line option
warning[W0001]: -o: expected a value, ignoring this option
warning[W0001]: --output: expected a value, ignoring this option
warning[W0001]: no input files were given
